process/contributing.md

```diff
@@ -8,7 +8,7 @@ Within open source there are multiple approaches to contribution and licensing,
 
 ## License
 
-Generally net new open source projects at The Linux Foundation leverage the [Apache License, Version 2.0](http://www.apache.org/licenses/LICENSE-2.0) for their codebase, a [Community Data License Agreement (CDLA) license](https://cdla.io/) for any data sets, and the [Creative Commons Attribution 4.0 International License](http://creativecommons.org/licenses/by/4.0/) for all documentation and non-code assets. These licenses are widely used and understood by both developers and organizations alike, providing flexibility for downstream usage and patent protection for those contributing code.
+Generally net new open source projects at The Linux Foundation leverage the [Apache License, Version 2.0] for their codebase, a [Community Data License Agreement (CDLA) license] for any data sets, and the [Creative Commons Attribution 4.0 International License] for all documentation and non-code assets. These licenses are widely used and understood by both developers and organizations alike, providing flexibility for downstream usage and patent protection for those contributing code.
```

Comment:
Should this say "net new open source projects at the Academy Software Foundation leverage..."?

Reply:
We certainly could. I recall the background for saying 'Linux Foundation' here was to establish that this was broader practice within our hosted projects, as at the time this was written there was only OpenVDB. I'd also share that most ASWF projects are using a BSD-3-clause license since they were historically using that license.

Reply:
"net new"?

Reply:
"net new" = projects without a license before

Reply:
Then let's say "projects that did not have a license before". I think that others, like me, will think "net new" is a typo.

Reply:
Like you mistyped new as "net", and then when you went to fix it, you added "new" instead of changing the existing "net." That's what I thought had happened here.

Reply:
Good suggestion - fixed now.